packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:html';
+
+class DragScroll {
+  var mousemove = 'mousemove';
+  var mouseup = 'mouseup';
+  var mousedown = 'mousedown';
+  var EventListener = 'EventListener';
+
+// ??? what are these ???
+//  var addEventListener = 'add'+EventListener;
```

Comment:
these are the addEventListener and removeEventListeners on the Dart element class
you can use those directly or use the Dart stream listen api instead if you want. doesn't matter it is the same thing.